src/registry.test.ts

```diff
@@ -56,13 +56,20 @@ describe("getEntry", () => {
 });
 
 describe("getEntryUrl", () => {
+  // Hard-code public key and expected encoded values to catch any breaking changes to the encoding code.
+  const publicKey = "c1197e1275fbf570d21dde01a00af83ed4a743d1884e4a09cebce0dd21ae254c";
+  const encodedPK = "ed25519%3Ac1197e1275fbf570d21dde01a00af83ed4a743d1884e4a09cebce0dd21ae254c";
+  const encodedDK = "7c96a0537ab2aaac9cfe0eca217732f4e10791625b4ab4c17e4d91c8078713b9";
+
   it("should generate the correct registry url for the given entry", () => {
-    // Hard-code public key and expected encoded values to catch any breaking changes to the encoding code.
-    const publicKey = "c1197e1275fbf570d21dde01a00af83ed4a743d1884e4a09cebce0dd21ae254c";
-    const encodedPK = "ed25519%3Ac1197e1275fbf570d21dde01a00af83ed4a743d1884e4a09cebce0dd21ae254c";
-    const encodedDK = "7c96a0537ab2aaac9cfe0eca217732f4e10791625b4ab4c17e4d91c8078713b9";
     const url = client.registry.getEntryUrl(publicKey, dataKey);
 
     expect(url).toEqual(`${portalUrl}/skynet/registry?publickey=${encodedPK}&datakey=${encodedDK}`);
   });
+
+  it("should trim the prefix if it is provided", () => {
+    const url = client.registry.getEntryUrl(`ed25519:${publicKey}`, dataKey);
```

Comment:
I'm wondering if we need to test for `encodedPK` potentially being passed in here ('%3A' instead of ':'). Maybe if someone is doing in-browser URL parsing?